lib/total.rb

```diff
@@ -0,0 +1,26 @@
+require 'dishes'
+require 'calculator'
+require 'order'
+
+class Total < Calculator
```

Comment:
It feels like maybe the distribution of responsibilities amongst Total, Calculator, Order could be rethought.  Let's have a chat about this.  Can you grab me when you have a second?